### ISSUE TITLE
Code Recommendations Not Working

### ISSUE THREAD

**Issue by TheHomie8672** [TheHomie8672/RedBaron#2]
Th code recommendation function is supposed to extract the contents on screen, identify any code on screen, send the code through the Divinci-codex engine, and return suggestions on the code. It does not, however, do such a thing, because that would be too convenient. 

-Get_Code_Recomendations (line 81) 

I just looked and some code seems to be missing.

**Comment by Moon1570** [comment]
Hi. Can I work on this issue? I want to learn and contribute.

**Comment by TheHomie8672** [comment]
Absolutely, I'm sorry for the late response, I genuinely didn't expect anyone to take interest. I'd be happy to collab!
